<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui</class>
 <widget class="QMainWindow" name="gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Symulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="mainGridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QChartView" name="chartSetpointOutput">
      <property name="title">
       <string>Wartosc zadana i regulowana</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QChartView" name="chartError">
      <property name="title">
       <string>Wykres Uchybow</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QChartView" name="chartControl">
      <property name="title">
       <string>Wykres Sterowania</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="pidLayout">
      <item>
       <widget class="QLabel" name="pidLabel">
        <property name="text">
         <string>PID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="kpInput">
        <property name="placeholderText">
         <string>Kp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="kiInput">
        <property name="placeholderText">
         <string>Ki</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="kdInput">
        <property name="placeholderText">
         <string>Kd</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QVBoxLayout" name="arxLayout">
      <item>
       <widget class="QLabel" name="arxLabel">
        <property name="text">
         <string>ARX</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="arxAInput">
        <property name="placeholderText">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="arxBInput">
        <property name="placeholderText">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="arxDelayInput">
        <property name="placeholderText">
         <string>Opoznienie</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLCDNumber" name="simulationTimeDisplay">
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="startStopButton">
      <property name="text">
       <string>Start / Stop</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QTextBrowser" name="logOutput">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd"&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name="qrichtext" content="1" /&gt;&lt;style type="text/css"&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;"&gt;
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="maximumHeight">
       <number>100</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>